github/Issue.py

```diff
@@ -230,6 +230,17 @@ def user(self):
         self._completeIfNotSet(self._user)
         return self._user.value
 
+    def as_pull_request(self):
+        """
+        :calls: `GET /repos/:owner/:repo/pulls/:number <http://developer.github.com/v3/pulls>`_
+        :rtype: :class:`github.PullRequest.PullRequest`
+        """
+        headers, data = self._requester.requestJsonAndCheck(
+            "GET",
+            "/pulls/".join(self.url.rsplit("/issues/", 1))
```

Comment:
This is the hack to replace only last occurrence of the pattern; just `self.url.replace("/issues/", "/pulls/")` will replace all, which might be user name or repo name.
https://stackoverflow.com/questions/2556108/rreplace-how-to-replace-the-last-occurrence-of-an-expression-in-a-string

Reply:
Could there be more than one occurrence? If not, why not just use `str.replace()`

Reply:
I don't know much about the Github specification, but I cannot guarantee that following situation never happen:

- User/organization name or repository name is `"issues"` or `"pulls"`
- Hostname of Github Enterprise is configured to something like `"mycompany.com/issues/ghe"`

Of course it's ridiculous idea to do that, but if it's POSSIBLE, we should care that.

If we can say this never happens, I will use `str.replace()`.

Reply:
You are absolutely right. Thanks for being so prudent.